### DISCUSSION TITLE
欢迎使用现代化的新维加斯Mod指南，有任何问题或想法可以在这里讨论

### DISCUSSION THREAD

**Discussion by feelbetterhua** [feelbetterhua/nvguideline_cn#8] [General]
百度贴吧原帖：https://tieba.baidu.com/p/6204813459?
整个项目还是在进行中，只要Mod还在更新，指南就不会停止

**Comment by gitlidesheng** [comment]
听我说谢谢你，因为有你，温暖了四季

**Comment by TIAIMM** [comment]
我最近也想做一个基于VNV的新维加斯Modding指南，有无合作兴趣？

**Reply by feelbetterhua** [reply]
抱歉没有注意到你的消息，我也很久没有更新这个项目了，有什么我可以帮助的？

**Reply by TIAIMM** [reply]
我在写一个配置教程，目前整好了基础部分：https://zhuanlan.zhihu.com/p/574181412
